yarn.lock

```diff
@@ -17215,7 +17215,7 @@ rc@^1.2.7:
     minimist "^1.2.0"
     strip-json-comments "~2.0.1"
 
-react-autosuggest@9.4.3, react-autosuggest@^9.4.3:
+react-autosuggest@9.4.3:
```

Comment:
Can you send the result of `yarn why react-autosuggest` ? I'm surprised to always see this `9.4.3` version of react-autosuggest within the lock file.

Reply:
It is from cozy-bar in mespapiers-lib

```
yarn why v1.22.19
[1/4] 🤔  Why do we have the module "react-autosuggest"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "react-autosuggest@10.1.0"
info Reasons this module exists
   - "_project_#cozy-sharing" depends on it
   - Hoisted from "_project_#cozy-sharing#react-autosuggest"
info Disk size without dependencies: "248KB"
info Disk size with unique dependencies: "972KB"
info Disk size with transitive dependencies: "1.07MB"
info Number of shared dependencies: 9
=> Found "cozy-bar#react-autosuggest@9.4.3"
info This module exists because "_project_#cozy-mespapiers-lib#cozy-bar" depends on it.
info Disk size without dependencies: "380KB"
info Disk size with unique dependencies: "728KB"
info Disk size with transitive dependencies: "1008KB"
info Number of shared dependencies: 9
✨  Done in 0.63s.
```

Reply:
OK thanks for the information 👍